src/components/top_bar/color_editor/color_editor.xml

```diff
@@ -0,0 +1,16 @@
+<templates>
+  <t t-name="o-spreadsheet-ColorEditor">
+    <div class="d-flex align-items-center">
+      <ColorPickerWidget
+        currentColor="currentColor"
+        toggleColorPicker.bind="onClick"
+        showColorPicker="isMenuOpen"
+        onColorPicked="(color) => this.setColor(color)"
+        title="textColor"
+        icon="props.icon"
+        dropdownMaxHeight="dropdownMaxHeight"
```

Comment:
We can probably remove all the `dropdownMaxHeight` props now that we use popover everywhere. Having it equal to gridHeight is not even correct now that the topbar can take multiple lines. Just let the popover handle it IMO, it will be limited to the o-spreadsheet div by default.